consensus/src/aggregation/mod.rs

```diff
@@ -0,0 +1,511 @@
+//! Aggregation module
```

Comment:
When writing this out, I suggest taking considerable space to explain why we don't handle resolution of missing signatures here.

Reply:
https://github.com/commonwarexyz/monorepo/pull/974/commits/c65eef2f69efc62cc55d64d5b7f546d44ed58b32